### DISCUSSION TITLE
Can the Drawer be closed automatically when clicking navigation links?

### DISCUSSION THREAD

**Discussion by 29thfloor** [saadeghi/daisyui#1256] [Q&A] [answered]
I'm trying to use the Drawer and Menu components as navigation (like the DaisyUI website) but the Drawer stays open when clicking on menu links to other pages. It seems like the DaisyUI website has the behavior I want. I'm not sure what I'm missing.

Was there some additional code needed to control the navigation behavior? I'm using SvelteKit, which is new to me, so I could be missing something there with the routing or what have you.

**Comment by saadeghi** [comment, marked as answer]
The checkbox rules if drawer should be open or closed. So, if you want to close the drawer when an item is clicked, you should control the value of checkbox using JS.  
On daisyUI website, I did the same.  
[Here](https://github.com/saadeghi/daisyui/blob/master/src/docs/src/routes/__layout.svelte) I bind the value of checkbox to a variable named `checked` and I have 2 functions for opening and closing.  
Then on each menu item [here](https://github.com/saadeghi/daisyui/blob/master/src/docs/src/components/Sidebar.svelte) I call my closeDrawer function on click.

Let me know if you have any questions

**Reply by 29thfloor** [reply]
Thank you!
